tests/roles/ceph_backend_configuration/tasks/main.yaml

```diff
@@ -2,7 +2,9 @@
   no_log: "{{ use_no_log }}"
   ansible.builtin.set_fact:
     ceph_backend_configuration_shell_vars: |
-      CEPH_SSH="{{ controller1_ssh }}"
+      # if tripleo uses external ceph, ssh to ceph nodes otherwise controller nodes
+      # for external ceph, external_ceph and ceph1_ssh vars should be defined
+      CEPH_SSH="{{ external_ceph | default(false) | ternary(ceph1_ssh, controller1_ssh) }}"
```

Comment:
This is useful for unidelta. In case of unidelta ceph is externally deployed, and we can reach it using `ceph1_ssh` instead of looking at the controller node that has no mons.
Backward compatibility is preserved by using ternary filter, and `external_ceph` is passed as an override only for unidelta based jobs.
I think I'm ok with this approach. I'll let @jistr @ciecierski comment on it and land it.

Reply:
Change looks fine to me. Do we have user-facing doc for externally deployed ceph explaining this case?

Reply:
Technically the same documentation applies, the only difference is the location where the ceph config files are found. For director deployed ceph we have them in the openstack controller nodes, but if ceph is externally deployed, we need to connect to those nodes. The rest of the assumptions are the same.

Reply:
@katarimanojk Thanks for pointer to documentation.